counterpartylib/lib/backend/__init__.py

```diff
@@ -110,7 +109,6 @@ def fee_per_kb(conf_target, mode, nblocks=None):
     return BACKEND().fee_per_kb(conf_target, mode, nblocks=nblocks)
 
 
-@functools.cache
```

Comment:
yeah I tested the Python cache but it's not better if not worse.. I don't think the size is the problem but the serialization to store it in the cache.. `bitcoinlib` returns classes .. probably with `dict` it would be faster.. I would look at this more closely by cleaning `get_tx_info()`.